Safiyya/circlemapping
# [Bug/465] Reorder network view options

### Issue
Fixes #465

### Description
Network view options needed to be corrected - i.e. "Connect all contributors" was showing the "Connect leads and contributors" view and vice versa. The default view is now "Show all connections" (`isAuthorityCentricMode = false` in the language of the code). This setting (along with others, e.g. map colour) is now persisted between after log out (reverting a change from March 2019 that cleared all settings from local storage).

### Setup
The README now includes instructions for copying production data into your local DB - that'll make it easy to test this locally without accessing the production database.

### Testing
1. Go to a map and open the network view in the RH side panel.
2. Open up network map options and check that "Show all connections" is selected and that the connections on the map have are not arrows.
3. Select "Connect leads and contributors" and check that the connections in the diagram are unidirectional arrows.
4. Change map colour.
5. Log out and log back in again.
6. Check that your choices of colour and network view option have been persisted.
